Python/sysmodule.c

```diff
@@ -2287,6 +2287,14 @@ sys_activate_stack_trampoline_impl(PyObject *module, const char *backend)
 /*[clinic end generated code: output=5783cdeb51874b43 input=a12df928758a82b4]*/
 {
 #ifdef PY_HAVE_PERF_TRAMPOLINE
+#ifdef _Py_TIER2
```

Comment:
The macro you want is `_Py_JIT`. `_Py_TIER2` also includes the tier 2 interpreter which is compatible with the trampolines.

Reply:
Thanks for the catch @Fidget-Spinner !